robot/Crisis-Management/resources/ContactPageObject.py

```diff
@@ -0,0 +1,22 @@
+from cumulusci.robotframework.pageobjects import DetailPage
+from cumulusci.robotframework.pageobjects import pageobject
+from BaseObjects import BaseCMPage
+from locators_48 import cm_lex_locators
```

Comment:
this should be "from CrisisManagement import cm_lex_locators"

Reply:
@skristem It can be both ways. It can be either `from locators_48 import cm_lex_locators`  or `from CrisisManagement import cm_lex_locators`

Reply:
there is a difference. If u look in CrisisManagement.py file cm_lex_locators will be updated based on api version of org where as if u r importing from a locator file directly, u will always be using that specific api version of locator file instead of choosing based on org

Reply:
Ah makes sense. Since SAL did not have it forgot about this API version logic.